utils/helpers.ts

```diff
@@ -1,2 +1,6 @@
 export const firstTextToUpperCase = (text: string) =>
   text[0].toUpperCase() + text.slice(1)
+
+export const getCount = (tasks: any, name: string) => {
+  return tasks?.filter((task) => task?.category == name)
+}
```

Comment:
Does it universal function? You should place only universal functions in helpers file or functions which are used in several different places.

Reply:
it is used twice